.golangci.yml

```diff
@@ -61,3 +61,5 @@ issues:
   max-per-linter: 0
   max-same-issues: 0
 
+skip-files:
+  - "example_.*.go"
```

Comment:
Is this a required/recommended format (vs `*_example.go`)?

Reply:
This was unnecessary, I'll remove it.